extensions/hawkbit-extension-artifact-repository-s3/src/main/java/org/eclipse/hawkbit/artifact/repository/S3Repository.java

```diff
@@ -220,4 +235,21 @@ private static DigestOutputStream openFileOutputStream(final File file, final Me
     private boolean exists(final String sha1) {
         return amazonS3.doesObjectExist(s3Properties.getBucketName(), sha1);
     }
+
+    @Override
+    public void deleteByTenant(final String tenant) {
+        final String folder = tenant.toUpperCase();
```

Comment:
`tenant.toUpperCase();` here is no `trim()` but in the `sanitizeTenant()` you trim the tenant, should be same behavior.